packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,144 @@
+| **size**          | `'xsmall' \| 'small' \| 'medium' \| 'large'`                     | `'small'`   | Specifies the size of the rendered Chips withing the ChipGroup                                                                 | ❌       |
+| **value**         | `string \|string[]`                                              | `undefined` | Acts as a controlled component by specifying the ChipGroup value                                                               | ❌       |
+| **intent**        | `'positive' \| 'negative' \| 'neutral'`                          | `'neutral'` | Sets the ChipGroups's visual variant, it will propagate down to all the Chips                                                  | ❌       |
+
+## Examples
+
+Here are a few illustrative examples showcasing the utilization of the `Chip` and `ChipGroup` components:
+
+### Basic/Uncontrolled Usage
+
+Single Chip selection:
+
+```jsx
+<Box>
+  <Text> Select Business type: </Text>
+  <ChipGroup defaultValue="proprietorship">
+    <Chip value="proprietorship">Proprietorship</Chip>
+    <Chip value="public">Public</Chip>
+    <Chip value="small-business">Small Business</Chip>
+  </ChipGroup>
+</Box>
+```
+
+Multiple Chip selection:
+
+```jsx
+<Box>
+  <Text> What other capabilities are you looking for? </Text>
+  <ChipGroup selectionType="multiple" defaultValue={['payment-links', 'offers']}>
+    <Chip value="payment-links">Automated Payment Links</Chip>
+    <Chip value="wallet">Wallet on My App</Chip>
+    <Chip value="offers">Offer discounts & EMI options</Chip>
+  </ChipGroup>
+</Box>
+```
+
+### Controlled Usage
+
+```jsx
+function Single() {
+  // string value when selectionType is 'single' (default)
+  const [selected, setSelected] = useState('proprietorship');
+
+  return (
+    <Box>
+      <Text> Select Business type: </Text>
+      <ChipGroup
+        selectionType="single"
+        value={selected}
+        onChange={({ values }) => setSelected(values[0])}
+      >
+        <Chip value="proprietorship">Proprietorship</Chip>
+        <Chip value="public">Public</Chip>
+        <Chip value="small-business">Small Business</Chip>
+      </ChipGroup>
+    </Box>
+  );
+}
+
+function Multiple() {
+  // array of strings value when selectionType is 'multiple'
+  const [selected, setSelected] = useState(['in-progress', 'pending']);
+
+  return (
+    <Box>
+      <Text> Quick Filters: </Text>
+      <ChipGroup
+        selectionType="multiple"
+        value={selected}
+        onChange={({ values }) => setSelected(values)}
+      >
+        <Chip value="refunded">Refunded</Chip>
+        <Chip value="failed">Failed</Chip>
+        <Chip value="pending">Pending</Chip>
+        <Chip value="in-progress">In Progress</Chip>
+      </ChipGroup>
+    </Box>
+  );
+}
+```
+
```

Comment:
Might be good to add a form submission based example as well

Reply:
I feel the controlled/uncontrolled usage examples should suffice.

Reply:
I meant from perspective of a form specifically so we can see in action what would it take for consumers to build forms this way. For eg. form values, matching the label styling, spacing et al.